vic-class12-assignment/gulpfile.js

```diff
@@ -0,0 +1,21 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const watch = require('gulp-watch');
+
+gulp.task('default', ['lint', 'test'], () => {
+  console.log('started');
+});
+
+gulp.task('lint', () => {
+  gulp.src('/server.js')
+  .pipe(eslint());
+  .pipe(eslint.format());
```

Comment:
Be careful here.  You cant pipe something after you have added a semi-colon.  The semi-colon on line 12 will prevent the pipe on line 13 from happening. I'm betting if you run this the way it is you will get an error that says "unexpected token ."  .